clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -792,11 +792,11 @@ public void testJoinGroupRequestVersion0RebalanceTimeout() {
 
     @Test
     public void testOffsetFetchRequestBuilderToString() {
-        String allTopicPartitionsString = OffsetFetchRequest.Builder.allTopicPartitions("someGroup").toString();
+        String allTopicPartitionsString = new OffsetFetchRequest.Builder("someGroup", false, null).toString();
 
         assertTrue(allTopicPartitionsString.contains("groupId='someGroup', topics=null"));
         String string = new OffsetFetchRequest.Builder("group1",
-            Collections.singletonList(new TopicPartition("test11", 1))).toString();
+            false, Collections.singletonList(new TopicPartition("test11", 1))).toString();
```

Comment:
Why do we use `false` (same below) -- should we test add test with `true`, too?

Reply:
Sounds good